aea/aea_builder.py

```diff
@@ -987,7 +987,7 @@ def from_aea_project(
         """
         aea_project_path = Path(aea_project_path)
         cls._try_to_load_agent_configuration_file(aea_project_path)
-        _verify_or_create_private_keys(aea_project_path)
+        verify_or_create_private_keys(aea_project_path)
```

Comment:
This is used only in this module here (not it is not the one from cli); so please revert